refreeze.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo 'Refreezing...'
+deepest_level=$(find . -name 'package.dhall' | sed 's/[^\/]//g' | awk '{ print length }' | sort | tail -n 1)
+for depth in $(seq "${deepest_level}" -1 1) ; do
+  while IFS= read -r -d '' package_dhall ; do
+    echo "Freezing: ${package_dhall} ..."
+    dhall freeze --cache --all --inplace "${package_dhall}"
+  done < <(find . -mindepth "${depth}" -maxdepth "${depth}" -name 'package.dhall' -print0)
```

Comment:
Same as above

Reply:
Log messages reformatted.
Here the double-loop is necessary so because the deepest levels do need to be frozen before shallower levels